### PR TITLE
Completed html-lists exercise.

### PR DESCRIPTION
**Pull request by Yang-Darren**
image

### CODE REVIEW

File: html-lists/index.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<body>
+  <h1>HTML Lists</h1>
+  <h2>Ordered List Example</h2>
+  <ol>
+    <li>Learn to code.</li>
+    <li>Build Projects.</li>
+    <li>Get a job.</li>
+  </ol>
+  <h2>Unordered List Example</h2>
+  <li>HTML</li>
```
Comment (BrettAlbright) [inline html-lists/index.html]: you are missing the `ul` element for the `li`s. make sure to include that element!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

File: html-lists/index.html (hunk 1 above)
Comment (Yang-Darren) [reply html-lists/index.html]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**